src/Defra.PTS.Web.UI/Views/Content/Help.cshtml

```diff
@@ -2,7 +2,7 @@
 @using Microsoft.AspNetCore.Mvc.Localization
 @inject IViewLocalizer Localizer
 @using Microsoft.Extensions.Localization;
-@inject IStringLocalizer<SharedResource> sharedlocalizer
+@inject IStringLocalizer<ISharedResource> sharedlocalizer
```

Comment:
Does the welsh still work?

Reply:
Looks good with Welsh confirmed

Reply:
Now works with latest pushed code :) 

![image](https://github.com/user-attachments/assets/96616e12-2041-4dfd-bc2a-786d04a3aac1)